### PR TITLE
Wiring animation with storage

### PR DESCRIPTION
**Pull request by anand-sundaram**
`AnimationManager` can now save newly created `AnimationType`s and retrieve them from datastore. 

I have also removed the hard-coded animations, so now animations only work after they have been set to pad.

### CODE REVIEW

File: MIDIchlorians/MIDIchlorians/AnimationManager.swift [hunk 1]
```diff
@@ -14,6 +14,13 @@ class AnimationManager {
     private var animationTypes = [String: AnimationType]()
 
     func getAllAnimationTypesNames() -> [String] {
+        let storedAnimationTypes = DataManager.instance.loadAllAnimationTypes()
+        _ = storedAnimationTypes.map({(animationTypeString: String) in
```
Comment (advaypal) [inline MIDIchlorians/MIDIchlorians/AnimationManager.swift]: Why use map if you aren't modifying the list at all? Wouldn't a normal for loop make more sense?

Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/AnimationManager.swift]: alternatively you can use `forEach`

Comment (anand-sundaram) [reply MIDIchlorians/MIDIchlorians/AnimationManager.swift]: @advaypal right you are

**Comment by anand-sundaram** [comment]
Updated based on comments by @advaypal and @ngzhian. Please merge if everything else looks fine!

**Pull request merged**